tests/dummy/app/pods/components/frost-create-form/template.hbs

```diff
@@ -0,0 +1,18 @@
+{{#frost-modal-input title='Test title' subtitle='Subtitle'}}
+{{!#frost-form-container titleComponent='myTitle'}}
+
+  {{frost-bunsen-form
+      model=bunsenModel
+      view=bunsenView
+      onChange=(action 'formValueChanged')
```

Comment:
We need to be consistent with our event names.  Currently we're using dasherized (e.g. on-change)